### PR TITLE
Feature/add requests

### PR DESCRIPTION
**Pull request by nairelyshz**

### CODE REVIEW

**Review by yuniquintero — APPROVED** [review]

**Review by martinezfabiola — APPROVED** [review]

File: src/js/services/requests.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+const axios = require('axios')
+
+module.exports = {
+  get_health: function () {
```
Comment (german1608) [inline src/js/services/requests.js]: Please use camel case naming. `getHealth`

File: src/js/services/requests.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return new Promise((resolve, reject) => {
+      axios.get(url).then(response => {
+        resolve((response.status))
+      }).catch(error => {
+        reject(error)
+      })
+    })
+  },
+
+  get_calculate_plain_text: function (queryParameters) {
```
Comment (german1608) [inline src/js/services/requests.js]: Camel case please

File: src/js/services/requests.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    return new Promise((resolve, reject) => {
+      axios.get(url).then(async (response) => {
+        resolve(response)
+      }).catch(function (error) {
+        reject(error)
+      })
+    })
+  },
+
+  get_calculate_twitter_user: function (userId) {
```
Comment (german1608) [inline src/js/services/requests.js]: Camel case please

File: src/js/services/requests.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    return new Promise((resolve, reject) => {
+      axios.get(url).then(async (response) => {
+        resolve(response)
+      }).catch(function (error) {
+        reject(error)
+      })
+    })
+  },
+
+  get_calculate_twitter_tweets: function (queryParameters) {
```
Comment (german1608) [inline src/js/services/requests.js]: Camel case please

**Review by german1608 — CHANGES_REQUESTED** [review]

**Review by german1608 — APPROVED** [review]

**Pull request merged**